rfc-001-announcement-channel-protocol.md

```diff
@@ -0,0 +1,28 @@
+# What content is applicable for the announcements channel
+
+## Summary
+
+What types of messages are relevant to the newly renamed announcements channel?
+
+## Problem
+
+Off the back of the Deep Work retro hosted by Chris, we had common consensus that #general too noisy.
+
+To get round that we've created a new channel, #common-room for any chatter, and renamed general to announcements.
+
+There is a lot of ambiguity about what content is suitable for the new channel.
+
+## Proposal
+
+All content in #announcements should be a notification/message that everyone should have visibility on.
+
+In order to keep the channel "clean" any replies should go into a thread for that post.
+
+Types of content for announcements;
+
```

Comment:
Can we also add organising made tech internal work that needs everyones :eyes: on it e.g. stuff for bizops goals

Reply:
What aspects of the goals, as we already have #bizOps is that not a more appropriate place to have those discussions?

Reply:
I'm thinking stuff that involves the rest of the team - but perhaps we can resolve this in another RFC